### PR TITLE
Version 4

### PR DESCRIPTION
**Pull request by i-like-robots**
This bumps all of the Origami dependencies to their latest versions and updates their Sass integrations where necessary.

The difference to the output of `main.scss` can [viewed here](https://www.diffchecker.com/pI9bdz5g).

I have removed most of the defined colour use cases as the majority are unused or are declared elsewhere.

### CODE REVIEW

File: colors/_functions.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @function getColor($args...) {
-	@return oColorsGetPaletteColor($args...);
+	@return oColorsByName($args...);
 }
 
 @function getColorFor($args...) {
-	@return oColorsGetColorFor($args...);
+	@return oColorsByUsecase($args...);
```
Comment (notlee) [inline colors/_functions.scss]: Good opportunity to get rid of `getColorFor` and `getColor`? 
`oColorsByUsecase` is different to `oColorsFor`, so there will be some 
migration steps to document here.

File: colors/_set-use-cases.scss [hunk 2]
```diff
@@ -1,58 +1,25 @@
...
-@include oColorsSetColor('fastft-highlight', #f5dfd9);
-
-// Article theme colors
-@include oColorsSetUseCase(article-standard, border, 'black-80');
-@include oColorsSetUseCase(article-standard, text, 'black-80');
-@include oColorsSetUseCase(article-standard, background, 'black-80');
-@include oColorsSetUseCase(article-analysis, border, 'black-80');
-@include oColorsSetUseCase(article-analysis, text, 'black-80');
-@include oColorsSetUseCase(article-analysis, background, 'black-80');
-@include oColorsSetUseCase(article-comment, border, 'oxford-30');
```
Comment (notlee) [inline colors/_set-use-cases.scss]: 🎉

File: colors/_functions.scss (hunk 1 above)
Comment (i-like-robots) [reply colors/_functions.scss]: Let's do it, rip the plaster off!

File: typography/main.scss [hunk 3]
```diff
@@ -1,26 +1,25 @@
...
+		'metric': (
+			('weight': 'regular', 'style': 'normal'),
+			('weight': 'semibold', 'style': 'normal'),
+			('weight': 'medium', 'style': 'normal'),
+		)
+	));
 
 	html {
-		//TODO - move elsewhere??
-		@include oColorsFor(page);
```
Comment (edds) [inline typography/main.scss]: This used to set background colours to pink. Did this get moved somewhere else?

Comment (i-like-robots) [reply typography/main.scss]: It's always been set by both `n-ui` and Page Kit so this was a duplicate!

https://github.com/Financial-Times/n-ui/blob/master/browser/layout/main.scss#L10-L13

https://github.com/Financial-Times/dotcom-page-kit/blob/master/packages/dotcom-ui-shell/src/components/Shell.tsx#L51-L57

**Pull request merged**